react-example/.eslintrc

```diff
@@ -5,7 +5,8 @@
   ],
   "rules": {
     "@typescript-eslint/no-empty-interface": "off",
-    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
```

Comment:
Adding `import/no-unresolved` at the react sample app level bc it's expecting `@iterable/web-sdk` to be in the dependencies.